ui/app/AppLayouts/Chat/panels/communities/JoinPermissionsOverlayPanel.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import AppLayouts.Chat.helpers 1.0
+import AppLayouts.Chat.controls.community 1.0
+
+import SortFilterProxyModel 0.2
+
+
+Control {
+    id: root
+
+    property bool joinCommunity: true // Otherwise it means join channel action
+    property bool requirementsMet: false
+    property bool isInvitationPending: false
+    property bool isJoinRequestRejected: false
+    property string communityName
+    property var communityHoldings
```

Comment:
usually for models we use names like `communityHoldingsModel` for readability, wdyt about using this convention here as well?